webapp/portlet/src/main/webapp/vue-apps/common/components/ActivityShareDrawer.vue

```diff
@@ -112,9 +114,16 @@ export default {
       this.description = '';
       this.sharing = false;
     },
-    open(activityId, currentApp) {
-      this.activityId = activityId;
+    open(activity, currentApp) {
+      this.activityId = activity.id;
       this.currentApp = currentApp;
+      if (typeof(activity.news)  !== 'undefined') {
```

Comment:
we can not mention News here , as it is an external addon. we need to manage to publish activities based on their content and unify how they will render depending on their contents.